integration_tests/test_string.py

```diff
@@ -0,0 +1,10 @@
+from string import (capitalize)
+from ltypes import i32, f64, f32, i64, c32, c64
+
+def test_capitalize():
```

Comment:
Thanks for the PR. You'd need to call this function -- and add this file to `run_tests.py`.